climakitae/data/variable_descriptions.csv

```diff
@@ -0,0 +1,22 @@
+WIND_MAG,Wind Magnitude at 10 m,Wind speed at 10 meters above the Earth's surface. Computed by taking the vector magnitude of the west-east (u) component of wind and the north-south (v) component of wind.
+U10,West-East component of Wind at 10m,"Wind coming from the west, measured at 10 meters above Earth's surface. By covention, wind coming from the east is negative."
+V10,North-South component of Wind at 10m,"Wind coming from the north, measured at 10 meters above Earth's surface. By convention, wind coming from the south is negative."
+PSFC,Surface Pressure,Air pressure at Earth's surface.
+SWDDIF,Shortwave surface downward diffuse irradiance,"The portion of the total incoming solar radiation that has been scattered (by the atmosphere, clouds, etc.) before reaching the Earth’s surface."
+SNOWNC,Snowfall (snow and ice),Total precipitation in the form of snow and ice. 
+Q2,2m Water Vapor Mixing Ratio,"The ratio of the mass of water vapor over the mass of dry air, measured 2 meters above Earth's surface."
+TSK,Surface skin temperature,Temperature of the Earth's surface (ground surface).
+LWDNB,Instantaneous downwelling longwave flux at bottom,Longwave radiation emitted by Earth's surface that is reflected back downwards by clouds. 
+LWDNBC,Instantaneous downwelling clear sky longwave flux at bottom,"Longwave radiation emitted by Earth's surface that is reflected back downwards by the atmosphere, excluding the effect of clouds."
+LWUPB,Instantaneous upwelling longwave flux at bottom,Total amount of longwave radiation outputted by Earth's surface. 
+LWUPBC,Instantaneous upwelling clear sky longwave flux at bottom,Longwave radiation emitted by Earth's surface that passes through the atmosphere (i.e. is not reflected back to the surface or absorbed by the atmosphere). 
+SWDNB,Instantaneous downwelling shortwave flux at bottom,"Shortwave radiation from the sun that reaches Earth's surface, minus radiation reflected back to space by clouds or absorbed by the atmosphere."
+SWDNBC,Instantaneous downwelling clear sky shortwave flux at bottom,Shortwave radiation from the sun that reaches Earth's surface.
+SWUPB,Instantaneous upwelling shortwave flux at bottom,"Shortwave radiation from the sun that is reflected by Earth's surface, some of which may be reflected back downwards by clouds (i.e. not reflected back to space)."
+SWUPBC,Instantaneous upwelling clear sky shortwave flux at bottom,Shortwave radiation from the sun that is reflected by Earth's surface back to space.
+RAINC,Accumulated total cumulus precipitation,"Any form of precipitation (rain, snow, etc.) that is produced specifically by cumulus/cumulonimbus clouds. These are generally small clouds that bring heavy precipitation."
+RAINNC,Accumulated total grid scale precipitation,"Any form of precipitation (rain, snow, etc.) captured by the model resolution (i.e. 45km). Small clouds (including cumulus clouds) typically are excluded from this."
```

Comment:
Suggesting the following for even greater clarity and some parallelism between these options:
TOT_PRECIP, Precipitation (total), ...
RAINC, Precipitation (cumulus portion only), "Accumulated total cumulus precipitation. Any form..."
RAINNC, Precipitation (grid-scale portion only), "Accumulated total grid scale precipitation. Any form..."

keeping the latter two towards the bottom.

Also, change the order so that SWDDIF moves down near the other SW radiation variables.

Reply:
Awesome, I like these changes a lot. Thanks for the feedback. Just pushed those changes. RAINC and RAINCC are currently at the bottom of the dropdown. Let me know if you think they need to be moved up higher.
<img width="472" alt="Screen Shot 2022-07-22 at 4 15 08 PM" src="https://user-images.githubusercontent.com/66140951/180579599-8015d0fd-2470-424a-a7d9-411097586740.png">